kasa/discover.py

```diff
@@ -334,17 +343,21 @@ async def connect_single(
         :rtype: SmartDevice
```

Comment:
Would you mind updating the docstring ("the device type is discovered .." part) and adding the missing `:param:` s while you are at i t?

Reply:
Adjusted the doc string